spec/api.json

```diff
@@ -957,4270 +662,6796 @@
+        "summary": "Create workspace",
+        "tags": [
+          "workspaces"
+        ],
+        "parameters": [
+          {
+            "name": "orgId",
+            "description": "Organization ID",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "projectId",
+            "description": "Project ID",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "Authorization",
+            "description": "Access token",
+            "in": "header",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "x-api-key",
+            "description": "API key",
+            "in": "header",
+            "required": true,
+            "schema": {
+              "type": "string"
             }
-          ]
-        }
-        },
-    "/organizations/{orgId}/apps": {
-      "get": {
-        "summary": "get all Applications for a user",
-          "tags": [
-            "AppRegistry"
-          ],
-          "parameters": [
-            {
-              "name": "orgId",
-              "description": "AMS Organization ID",
-              "in": "path",
-              "required": true,
-              "schema": {
-                "type": "string"
-              }
-            },
-            {
-            "name": "appType",
-            "description": "Application type - use JGR",
-            "in": "query",
-              "required": true,
-              "schema": {
-                "type": "string"
-              }
-      },
-            {
-            "name": "offset",
-            "description": "offset",
-            "in": "query",
-              "required": true,
-              "schema": {
-              "type": "integer"
-              }
-            },
-            {
-            "name": "pageSize",
-            "description": "pageSize",
-            "in": "query",
-              "required": true,
-              "schema": {
-              "type": "integer"
-              }
-            },
-            {
-              "name": "Authorization",
-              "description": "Authorization",
-              "in": "header",
-              "required": true,
+          }
+        ],
+        "requestBody": {
+          "content": {
+            "application/json": {
               "schema": {
-                "type": "string"
+                "type": "object",
+                "required": [
+                  "name"
+                ],
+                "properties": {
+                  "name": {
+                    "type": "string"
+                  },
+                  "title": {
+                    "type": "string"
+                  },
+                  "description": {
+                    "ty
```

Comment:
Note: here I had to manually edit the swagger, in the original swagger definition the path parameter `{credentialType}` is called `{integrationType}` which mismatches with the required parameter definition below. We should keep this in mind when updating the swagger next time

Reply:
all good got updated in swagger spec